tsconfig.json

```diff
@@ -1,4 +1,5 @@
 {
+	"extends": "./node_modules/mol-lib-config/shared-config/tsconfig.json",
```

Comment:
this should be removed? We are not consuming this lib. Anyway can probably use the same one as the DS